core/storage/skill/gae_models.py

```diff
@@ -150,3 +150,30 @@ class SkillSummaryModel(base_models.BaseModel):
     # model was created).
     skill_model_created_on = ndb.DateTimeProperty(required=True, indexed=True)
     version = ndb.IntegerProperty(required=True)
+
+
+class SkillsMasteryModel(base_models.BaseModel):
+    """Mastery model of a user for Oppia Skill.
+
+    This model stores progress of each skill for a given user
+    and returns skill id.
+    """
+
+    # The user id of the currently logged in user.
+    user_id = ndb.StringProperty(required=True, indexed=True)
+    # The skill id for which degree of mastery is stored.
+    skill_id = ndb.StringProperty(required=True, indexed=True)
+    # The degree of mastery of the user in a skill.
+    degree_of_mastery = ndb.FloatProperty(required=True, indexed=True)
+
+    @classmethod
+    def get_user_mastery(cls, USER_ID):
```

Comment:
USER_ID is not constant, right? So, it would be better if it's in lower case.
Ditto below.